### ISSUE TITLE
No wegihts for loss functions

### ISSUE THREAD

**Issue by UdonDa** [PKU-ICST-MIPL/PosterLayout-CVPR2023#17]
Hi, @theKinsley!
Thank you for sharing your nice code!

I check your RecLoss.py, then I find that `self.weight_dict` is not used. So, current your code does not apply any weights for objectives.
https://github.com/PKU-ICST-MIPL/PosterLayout-CVPR2023/blob/main/RecLoss.py#L100

Actually, DETR applies the weight in the training loop.
https://github.com/facebookresearch/detr/blob/3af9fa878e73b6894ce3596450a8d9b89d918ca9/engine.py#L35

It should be applied [here](https://github.com/PKU-ICST-MIPL/PosterLayout-CVPR2023/blob/main/main.py#L80) or inside of SetCriterion.
Is this a mistake after your refactoring?

**Comment by theKinsley** [comment]
Hello UdonDa, thank you for reporting the issue.
Exactly, we adopt the partial code from DETR. The weights are as default (unchanged as original) but seems not applied appropriately, so they are actually 1:1:1 during our implementation in this version.
Thank you for pointing this out!
